config/nav/config.json

```diff
@@ -51,7 +51,8 @@
 	"radioRepeaterThresholds":
 	{
 		"signalStrengthCutOff": 30.0,
-		"lowSignalWaitTime": 3
+		"lowSignalWaitTime": 2,
+		"numIterationsUpdateRepeaterDropOdom": 5
```

Comment:
This name is really long and I still don't know what it is for based off of it's name.